efficientdet/dataset/create_coco_tfrecord.py

```diff
@@ -313,7 +316,7 @@ def _get_caption_annotation(image_id):
     else:
       return None
 
-  pool = multiprocessing.Pool()
+  pool = multiprocessing.Pool(FLAGS.num_threads)
```

Comment:
`multiprocessing.Pool` creates a pool of processes, not of threads. If a thread pool is desired, `multiprocessing.dummy.Pool` would be equivalent but with threads.

Reply:
Gotcha. Seems like multiprocessing.dummy.Pool cannot fully utilize multi-cores. Maybe keep it as multiprocessing.Pool for now.

Reply:
How about ```ThreadPoolExecutor```.

Reply:
Both are good to me. I will stick to multiprocessing.Pool in this PR, but we can change it later if needed.